src/app/hooks/use-network-options.test.tsx

```diff
@@ -12,20 +12,9 @@ describe("useNetworkOptions hook", () => {
 
 		const networks = result.current.networkOptions;
 
-		expect(networks).toEqual([
-			{
-				label: "ARK Mainnet",
-				value: "ark.mainnet",
-			},
-			{
-				label: "ARK Devnet",
-				value: "ark.devnet",
-			},
-			{
-				label: "Compendia Mainnet",
-				value: "compendia.mainnet",
-			},
-		]);
+		expect(networks).toContainEqual({ value: "ark.mainnet", label: "ARK Mainnet" });
+		expect(networks).toContainEqual({ value: "ark.devnet", label: "ARK Devnet" });
+		expect(networks).toContainEqual({ value: "compendia.mainnet", label: "ARK Mainnet" });
```

Comment:
This should be Compendia Mainnet, must be either a mistake in the DW or SDK.

Reply:
Fixed to use `network.coinName()` but `network.coin()` seems to return `ARK` from sdk

Reply:
It returns ARK for the coin because that is the parent coin.